tests/ttnn/unit_tests/operations/ccl/test_reduce_scatter_post_commit.py

```diff
@@ -650,17 +648,28 @@ def test_width_sharded_reduce_scatter_post_commit(
             ttnn.CoreRangeSet({ttnn.CoreRange(ttnn.CoreCoord(0, 0), ttnn.CoreCoord(4, 1))}),
             (32, 160),
             ttnn.CoreRangeSet({ttnn.CoreRange(ttnn.CoreCoord(0, 0), ttnn.CoreCoord(7, 4))}),
+            2,
         ),
         (
             (1, 1, 32, 1280),
             (32, 128),
             ttnn.CoreRangeSet({ttnn.CoreRange(ttnn.CoreCoord(0, 0), ttnn.CoreCoord(4, 1))}),
             None,
             None,
+            2,
+        ),
+        (
+            (1, 1, 32, 320),
+            (32, 32),
+            ttnn.CoreRangeSet({ttnn.CoreRange(ttnn.CoreCoord(0, 0), ttnn.CoreCoord(4, 1))}),
+            None,
+            None,
+            1,
         ),
     ),
 )
-@pytest.mark.parametrize("topology", [ttnn.Topology.Ring, ttnn.Topology.Linear])
+@pytest.mark.parametrize("topology", [ttnn.Topology.Linear])
+# @pytest.mark.parametrize("topology", [ttnn.Topology.Ring, ttnn.Topology.Linear])
```

Comment:
Should we put back in the Ring topology test now that the issue testing is done?

Reply:
Yes! Thanks for the catch. This was not intended to be a part of the PR.